docs/getting-started.md

```diff
@@ -250,18 +250,20 @@ loki:
       - url: http://localhost:3100/loki/api/v1/push
 
 tempo:
-  receivers:
-    jaeger:
-      protocols:
-        grpc: # listens on the default jaeger grpc port: 14250
-  push_config:
-    endpoint: localhost:55680
-    insecure: true  # only add this if TLS is not required
-    batch:
-      timeout: 5s
-      send_batch_size: 100
-    queue:
-      retry_on_failure: true
+  configs:
```

Comment:
i don't necessarily disagree with this, but it's going to create a lot of copypasta boilerplate when you want to create multiple pipelines.

does prometheus/loki allow for completely independent config per remote write target?  should we just emulate those systems and have the same pipeline that writes to multiple backends?

Reply:
I think that's a separate PR, actually. There's two features someone would want:

1. Configure distinct pipelines to send to distinct locations (this PR) 
2. Configure one pipeline to send to multiple locations (future PR)

The "distinct pipelines" is now supported in all of our subsystems with this PR, but the tracing subsystem still lacks a way to send spans to multiple endpoints, which I def want to also support without having people to use this `configs` block.